src/app/auth/onboarding/styles.tsx

```diff
@@ -18,4 +20,20 @@ export default StyleSheet.create({
     paddingLeft: 43,
     paddingRight: 44,
   },
+  subtext: {
+    color: colors.darkGrey,
+    marginLeft: 10,
+  },
+  h1: {
+    marginTop: 30,
+  },
+  body1: {
+    marginTop: 25,
```

Comment:
Usually spacing values will be in multiples of 2, so when you see an odd-numbered pixel value its usually a figma rounding error and you should round them up. So here you can change this to margin top = 26